internal/cli/testdata/apko.lock.json

```diff
@@ -39,7 +39,7 @@
           "checksum": "sha1-cs+Hlyu5sY+1mmwKPedcRWj8E24="
         },
         "data": {
-          "range": "bytes=965-2899",
+          "range": "bytes=965-3864",
```

Comment:
@jonjohnsonjr these you use a careful sanity check.  I have no idea how things worked before

Reply:
Neither of these values looks right to me 🤔

Reply:
I think this needs to be using Size instead of InstalledSize. Previously it was using InstalledSize. Yours is using InstalledSize + Data Offset 🤔

Reply:
@jonjohnsonjr I believe this was the number you said on the call now, so PTAL and let me know what you think!